src/pages/UserHome.jsx

```diff
@@ -109,35 +107,40 @@ export default function UserHome() {
             Add Scholarship
           </Button>
         </Grid>
+        {scholarships.length === 0 && (
+          <Grid container className={classes.bannerRoot}>
+            <Grid item xs={3}>
+              <InboxIcon style={{ fontsize: 600 }} />
+            </Grid>
+            <Grid item>
+              <Typography variant="h5" gutterButtom>
+                No Scholarships Added Yet{' '}
+              </Typography>
+              <Typography
+                variant="contained"
+                color="primary"
+                component={Link}
+                to="/scholarships/new">
+                Add Scholarship
+              </Typography>
+            </Grid>
```

Comment:
These should be vertically centered. Take a look at the [Grid](https://material-ui.com/components/grid/) docs to learn how to do this. The [Interactive](https://material-ui.com/components/grid/#interactive) section might be helpful.